src/robot_module/src/robot_module/image_sub.py

```diff
@@ -0,0 +1,18 @@
+import rospy
+from .subscriber import ServerSub
+
+class ImageSub(ServerSub):
+    """
+    Subscriber which receives image data from a ROS camera.
+    """
+
+    def __init__(self, topic):
+        super().__init__(topic, CompressedImage)
+        self.image_cache = [] # TODO: Cache last 10 frames
```

Comment:
Curiosity: is there currently a demand for a cache by the task code? Or any significance behind the 10 frame limit?
Not saying this shouldn't be here, just wondering :)

Reply:
:D We'll be looking into camera code for the future patch
This file shouldn't be pushed into the main branch :S